src/harness/security_testcase.py

```diff
@@ -66,7 +67,14 @@ def SasReset(self):
     """Resets the SAS UUT to its initial state."""
     self._sas_admin.Reset()
 
-  def assertTlsHandshakeSucceed(self, base_url, ciphers, client_cert, client_key):
+  def getCertFilename(self, cert_name):
+    """Returns the absolute path of the file corresponding to the given |cert_name|.
+    """
+    harness_dir = os.path.dirname(
+                  os.path.abspath(inspect.getfile(inspect.currentframe())))
+    return os.path.join(harness_dir, 'certs', cert_name)
+
+  def assertTlsHandshakeSucceed(self, base_url, ciphers, client_cert, client_key,IsVerifyServercert=False):
```

Comment:
revert to the original
def assertTlsHandshakeSucceed(self, base_url, ciphers, client_cert, client_key)
?